frontend-student/src/components/coursePage/CommentsSection.jsx

```
@@ -0,0 +1,307 @@
+  const isCommentOwner = (comment) => {
+    return comment.isOwner || (comment.userId && comment.userId === currentUserId);
+  };
+
+  return (
+    <div className='mt-6 bg-white rounded-lg shadow-lg overflow-hidden border border-gray-200'>
+      {/* Header with toggle */}
+      <div
+        className='flex justify-between items-center p-4 bg-gradient-to-r from-blue-600 to-blue-500 text-white cursor-pointer'
+        onClick={toggleCollapse}
+      >
+        <div className='flex items-center space-x-2'>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)